<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register_maintenance</class>
 <widget class="QWidget" name="register_maintenance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>register_miantenance</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>住户编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>维修描述</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="report_dateEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>上报时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>维修时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>维修金额</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>是否从维修基金支出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>账号_id</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="repair_dateEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="resident_id_lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="description_textEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="yes">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>是</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="payment_lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>维修人_id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="repair_id">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
